contracts/colony/Colony.sol

```diff
@@ -72,6 +72,10 @@ contract Colony is ColonyStorage, PatriciaTreeProofs {
     return executeCall(_to, 0, _action);
   }
 
+  function annotateTransaction(bytes32 _txHash, string memory _metadata) public stoppable {
+    emit Annotation(_txHash, msg.sender, _metadata);
```

Comment:
As discussed, these arguments should be reordered (msg.sender coming first, and named agent in anticipation of #911